### PR TITLE
Update README with new setup

### PR DESCRIPTION
**Pull request by rossjones**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 ## Developer Setup
 
-Currently only Postgres and ElasticSearch are installed inside the VM - this is
-due to issues with virtualbox noticing file changes.  Because it doesn't notice
-file changes, live reloading during development doesn't work, and we like
-live-reloading. For now to develop on the host this means there are two stages.
+This repository currently uses the Virtual Machine specified by https://github.com/datagovuk/dgu_deploy
```
Comment (maxf) [inline README.md]: I think 'currently requires' is more accurate

File: README.md [hunk 2]
```diff
@@ -91,3 +66,15 @@ mix phoenix.server
...
+## Users
+
+Until authentication is addressed you must perform a manual step to set up your users in the prototype.
+
+1. Connect to your new CKAN from the dgu_deploy repository, and register some users and assign to the appropriate organisations.
+
+2. For each user, visit their user profile and obtain their API Keys.
+
+3. In the dguweb_dev database you need to assign these APIkeys to users who have registered in the prototype front-end.
+
```
Comment (maxf) [inline README.md:79]: Maybe it doesn't matter so much for now, but maybe we need to be more specific and name db tables or fields, etc?

File: README.md [hunk 3]
```diff
@@ -91,3 +66,15 @@ mix phoenix.server
...
+## Users
+
+Until authentication is addressed you must perform a manual step to set up your users in the prototype.
+
+1. Connect to your new CKAN from the dgu_deploy repository, and register some users and assign to the appropriate organisations.
+
+2. For each user, visit their user profile and obtain their API Keys.
+
+3. In the dguweb_dev database you need to assign these APIkeys to users who have registered in the prototype front-end.
```
Comment (maxf) [inline README.md]: 'API keys' (whitespace)
&lt;/pedantic&gt;

**Comment by rossjones** [comment]
ping @maxf

**Pull request merged**